README.md

```diff
@@ -18,7 +18,40 @@ None.
 
 | Variable | Description | Default | Required |
 |----------|-------------|---------|----------|
+| clamav_clamd_conf | A dictionary of values to set in the clamd configuration file. | `{}` | No |
+| clamav_configuration_backup | Whether or not to backup configuration files before changing. | `false` | No |
 | clamav_cron_frequency | The frequency of ClamAV scanning.  Must be one of: `hourly`, `daily`, `weekly`, or `monthly`. | `weekly` | No |
+| clamav_freshclam_conf | A dictionary of values to set in the freshclam configuration file. | `{}` | No |
```

Comment:
Since we have `clamav_configuration_backup` I would rather use the long form for consistency.

```suggestion
| clamav_freshclam_configuration | A dictionary of values to set in the freshclam configuration file. | `{}` | No |
```